File: src/main/java/com/paragon/client/managers/alt/AltManager.java
package com.paragon.client.managers.alt;

import java.util.ArrayList;

public class AltManager {

    private final ArrayList<Alt> alts = new ArrayList<>();

    public AltManager() {

    }

    public void addAlt(Alt alt) {
        this.alts.add(alt);
    }

    public void removeAlt(Alt alt) {
        alts.remove(alt);
    }

    public ArrayList<Alt> getAlts() {
        return alts;
    }

}

File: src/main/java/com/paragon/client/managers/alt/AltManager.kt
package com.paragon.client.managers.alt

/**
 * @author SooStrator1136
 */
class AltManager {

    val alts = ArrayList<Alt>()

    fun addAlt(alt: Alt) {
        alts.add(alt)
    }

    fun removeAlt(alt: Alt?) {
        alts.remove(alt)
    }

}